Core/UserDefaultsPropertyWrapper.swift

```diff
@@ -56,15 +56,25 @@ public struct UserDefaultsWrapper<T> {
 
     private let key: Key
     private let defaultValue: T
+    private let setIfEmpty: Bool
 
-    public init(key: Key, defaultValue: T) {
+    public init(key: Key, defaultValue: T, setIfEmpty: Bool = false) {
```

Comment:
This is for all properties that are by default initialized by experiments. 

Reason: if we don't set default here, but only read it every time this is fetched, then once experiment is over we will have to either:
- Migrate all users that did not set this option themselves to new default (and possibly annoy them).
- Based on the variant populate this property to avoid breaking UX for these people.

This option solves that problem as default can be now stored in user defaults on first access.